### ISSUE TITLE
No way to select language

### ISSUE THREAD

**Issue by gregersn** [Soundux/soundux.github.io#169]
The website is show to me in two languages,
one on the left side menu, and one on the main page, but I can see no button to choose which language I want.

It probably has to do partially with it automatically choosing language based on my browser settings, but there should still be an option to override it.

**Comment by D3SOX** [comment]
Yes, it currently uses the `navigator.language` from your browser.

**Comment by gregersn** [comment]
The menu is in English, while the main part of the site is in Norwegian. 

![Screenshot from 2022-02-10 15-04-54](https://user-images.githubusercontent.com/1364443/153423785-783b4e81-0e1d-4037-9430-449336c24874.png)

**Comment by D3SOX** [comment]
Oh, maybe I never added the option to translate the menu items

**Comment by D3SOX** [comment]
Yes they are hardcoded [here](https://github.com/Soundux/soundux.github.io/blob/master/src/layouts/default.vue#L86-L112)
